### PR TITLE
Feature/custom arrow and bug fixes

### PR DESCRIPTION
**Pull request by mertceyhan**
Fixing issue; #8 , #9 , #10

### CODE REVIEW

File: library/src/main/java/com/trendyol/showcase/showcase/ShowcaseManager.kt [hunk 1]
```diff
@@ -58,13 +58,13 @@ data class ShowcaseManager private constructor(
...
 
         return showcaseModel.copy(
-            titleTextColor = typedArray.getColor(R.styleable.Showcase_Theme_titleTextColor, showcaseModel.titleTextColor),
-            descriptionTextColor = typedArray.getColor(R.styleable.Showcase_Theme_descriptionTextColor, showcaseModel.descriptionTextColor),
-            closeButtonColor = typedArray.getColor(R.styleable.Showcase_Theme_closeButtonColor, showcaseModel.closeButtonColor),
-            popupBackgroundColor = typedArray.getColor(R.styleable.Showcase_Theme_popupBackgroundColor, showcaseModel.popupBackgroundColor),
-            windowBackgroundColor = typedArray.getColor(R.styleable.Showcase_Theme_windowBackgroundColor, showcaseModel.windowBackgroundColor),
-            showCloseButton = typedArray.getBoolean(R.styleable.Showcase_Theme_showCloseButton, showcaseModel.showCloseButton),
-            cancellableFromOutsideTouch = typedArray.getBoolean(R.styleable.Showcase_Theme_cancellableFromOutsideTouch, showcaseModel.cancellableFromOutsideTouch)
+                titleTextColor = typedArray.getColor(R.styleable.Showcase_Theme_titleTextColor, showcaseModel.titleTextColor),
```
Comment (erkutaras) [inline library/src/main/java/com/trendyol/showcase/showcase/ShowcaseManager.kt:61]: Different indentation has been used. Can you change 8 with 4 for the project?

File: library/src/main/java/com/trendyol/showcase/ui/tooltip/TooltipViewState.kt [hunk 2]
```diff
@@ -2,22 +2,26 @@ package com.trendyol.showcase.ui.tooltip
 
 import android.view.View
 import androidx.annotation.ColorInt
+import androidx.annotation.DrawableRes
+import com.trendyol.showcase.R
+import com.trendyol.showcase.util.Constants
 
 data class TooltipViewState(
```
Comment (selimtoksal) [inline library/src/main/java/com/trendyol/showcase/ui/tooltip/TooltipViewState.kt:9]: can you add unit tests for new functions

File: library/src/main/java/com/trendyol/showcase/ui/tooltip/TooltipViewState.kt [hunk 3]
```diff
@@ -30,11 +34,17 @@ data class TooltipViewState(
...
 
-    fun getTopArrowVisibility() =
-        if (arrowPosition == ArrowPosition.UP) View.VISIBLE else View.GONE
+    fun getTitleVisibility() = titleText.isNotEmpty()
 
-    fun getBottomArrowVisibility() =
-        if (arrowPosition == ArrowPosition.DOWN) View.VISIBLE else View.GONE
+    fun getDescriptionVisibility() = descriptionText.isNotEmpty()
+
+    fun getTopArrowResource() = if (arrowResource == Constants.DEFAULT_ARROW_RESOURCE) R.drawable.ic_arrow_up else arrowResource
```
Comment (selimtoksal) [inline library/src/main/java/com/trendyol/showcase/ui/tooltip/TooltipViewState.kt:41]: can you add unit test for new functions

File: library/src/main/java/com/trendyol/showcase/showcase/ShowcaseManager.kt (hunk 1 above)
Comment (mertceyhan) [reply library/src/main/java/com/trendyol/showcase/showcase/ShowcaseManager.kt]: Fixed

File: library/src/main/java/com/trendyol/showcase/ui/tooltip/TooltipViewState.kt (hunk 3 above)
Comment (mertceyhan) [reply library/src/main/java/com/trendyol/showcase/ui/tooltip/TooltipViewState.kt:41]: Unit test class for TooltipViewState created. You can check on the last commit

**Pull request merged**